app/assets/javascripts/discourse/tests/acceptance/dismiss-notification-modal-test.js

```diff
@@ -29,13 +29,12 @@ acceptance("Dismiss notification confirmation", function (needs) {
     await visit("/");
     await click(".current-user");
     await click(".notifications-dismiss");
-    assert.ok(exists(".bootbox.modal"));
+    assert.ok(exists(".dismiss-notification-confirmation"));
 
     assert.strictEqual(
-      query(".bootbox.modal .modal-body").innerText,
-      "You have 2 important notifications, are you sure you would like to dismiss?"
+      query(".dismiss-notification-confirmation-modal .modal-body").innerText,
+      "Are you sure? You have 2 important notifications."
```

Comment:
I am going to be annoying, but you should use the I18n.t helpers in the tests as well, otherwise this test will break if someone updates that text

Reply:
Great! I hadn't noticed that, thanks!

Reply:
Hmm, thinking about it, shouldn't the test break if someone updates that text? If the implementation states to use a key that doesn't exist, it will show up as jibberish and the test won't catch it.

Reply:
As spoken, I'll make the change for consistency!